_maps/WariosBattleCanyon/Wario's Battle Canyon.yaml

```diff
@@ -0,0 +1,177 @@
+---
```

Comment:
can you remove special characters from the yaml file name. Spaces are fine, though the ' needs to go